dev-system/support/run-kondo.sh

```diff
@@ -9,35 +9,67 @@
 #     run: ./dev-system/support/run-kondo.sh >> report.log
 
 # error or warning
-fail_level=error
+fail_level=warning
```

Comment:
What will change here?  Oh, now will a PR fail a check on warnings and not just errors?

Reply:
Yes, for all of the projects that are being linted now we will fail the script for any warnings.